source/whats-new.txt

```diff
@@ -47,12 +47,49 @@ version of {+driver-short+} will raise the minimum {+mdb-server+} version from
 What's New in 3.4
 -----------------
 
-The 3.4 driver release includes the following new features:
+The 3.4 driver release includes the following new features,
+improvements, and fixes:
 
 - Adds the ``Mql.Convert()`` LINQ method to convert between types when
   performing aggregations. To learn more, see the
   :ref:`csharp-linq-convert` section of the LINQ guide.
 
+- Adds support for the ``KeyValuePair.Create()`` LINQ method.
+
+- Add the ``GeoNear()`` aggregation builder method to construct a
+  ``$geoNear`` aggregation stage. 
+
+- Supports casting from an interface to a type that implements that
+  interface in LINQ queries.
+
+- Supports the ``Dictionary`` constructor in LINQ.
+
+- Adds the following options for Atlas Search queries:
+
+  - ``MatchCriteria`` in ``SearchTextOptions`` to set match critera for
+    terms passed to the ``Text()`` operator method.
+
+  - ``Synonyms`` in ``SearchPhraseOptions`` to set the name of the
+    synonym mapping definition when using the ``Phrase()`` operator
+    method.
+
+  To learn more about performing Atlas Search queries, see
+  the :ref:`csharp-atlas-search` guide.
+
+- Supports LINQ queries after casting ``IQueryable<Derived>`` objects to
+  ``IQueryable<Base>``.
+
+- Corrects Atlas Search operator behavior to use registered serializers
+  instead of predefined serializers. To implement the previous
+  behavior, set the following option: 
+
+  .. code-block:: csharp
+
+     SearchDefinition.UseConfiguredSerializers(false)
```

Comment:
S: because i'm sure someone will copy this into their app

```suggestion
     SearchDefinition.UseConfiguredSerializers(false);
```

Reply:
good call